src/io/internal/pipelines/dicom/structured-report-to-html.cxx

```diff
@@ -0,0 +1,510 @@
+/*
+ *
+ *  Copyright (C) 2000-2016, OFFIS e.V.
```

Comment:
@thewtex Do we want to add our own license header above or below this header? I assume the licenses be compatible?

Reply:
Yes, let's add it above